script.js

```diff
@@ -1,8 +1,14 @@
+/* eslint-disable no-cond-assign */
+/* eslint-disable no-plusplus */
+/* eslint-disable no-constant-condition */
+/* eslint-disable quotes */
+/* eslint-disable array-callback-return */
+/* eslint-disable no-unused-vars */

```

Comment:
[OPTIONAL] I understand it can be frustrating for you not to be able to solve some linter errors but I would advise you strongly do them because disabling linters is not a good practice and it useless the work of eslint in the file.